exchanges/gateio/gateio_wrapper.go

```diff
@@ -633,6 +633,10 @@ func (g *Gateio) UpdateTickers(ctx context.Context, a asset.Item) error {
 		var tickers []FuturesTicker
 		var ticks []FuturesTicker
 		for _, settle := range settlementCurrencies {
+			if a == asset.DeliveryFutures && !settle.Equal(currency.USDT) {
```

Comment:
We can see what you're doing.
Needs a comment to explain why you're doing it :)

Reply:
Done for this and the others 👍